restli-tools/src/main/resources/apiVmTemplates/action.vm

```diff
@@ -0,0 +1,114 @@
+#*
+Copyright (c) 2021 LinkedIn Corp.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*#
+
+  #foreach($method in $spec.actions)
+    #set($actionParamClassName = "${util.nameCapsCase($method.name)}ActionParameters")
+    #doc($method.schema.doc)
+
+    ## methods are named as "action<ActionName>" so it is consistent with current RequestBuilder naming
```

Comment:
I don't think we need the "action" prefix. 

Actions are usually  "verbs" and so client.<actionName> would be easier to read. For example,
greetings.purge(123)
vs
greetings.actionPurge(123)